src/libraries/Microsoft.Extensions.Logging/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,41 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please remove the suppression file now that we use an APICompat that loads references during baseline validation. You might need to rebase / merge with main.